### PR TITLE
V1 init

### PR DESCRIPTION
**Pull request by alavers**
@mspensieri @jpjoyal @jmtremblay @jugarrit @lemieux

### CODE REVIEW

File: source/rest/index.md [hunk 1]
```diff
@@ -327,6 +327,101 @@ When a webhook is created, a shared secret will be generated for it. The secret
...
+
+An `app` object is also returned which includes metadata about the app, such as information about push notification channels.
+
+| **Arguments**               |   |
+|-----------------------------|---|
+| **device**<br/>*required* | A descriptor of the user's device. See below. |
+| **userId**<br/>*optional* | A unique identifier for the app user. Unlike the `appUserId` which is generated by SupportKit, the `userId` is chosen by the API consumer. The `userId` can be used to link a user to the same conversation across multiple devices. [More information](/#users-on-multiple-devices)|
+
+<aside class="warning">
+**Caution:** If you're specifying a `userId` then in order to keep conversations private we strongly suggest [authenticating your users](/#authenticating-users-optional). If a `userId` is used without a JWT credential, then anyone who can discover a user's `userId` could potentially eavesdrop on the conversation.
```
Comment (jpjoyal) [inline source/rest/index.md]: wow this link actually works

File: source/rest/index.md [hunk 2]
```diff
@@ -327,6 +327,101 @@ When a webhook is created, a shared secret will be generated for it. The secret
...
+</aside>
+
+## Device
+
+Provided `userId`s are specified in the call to `/v1/init`, devices and app users have a many-to-many relationship. A single `userId` may use SupportKit from multiple devices and multiple different `userId`s may log in to the same device. Note however that only one `userId` can be active on a given device at a time. That is, if a device is shared among multiple `userId`s, only the most recent `userId` to have called `/v1/init` will receive push notifications from the integrated application on that device.
```
Comment (jpjoyal) [inline source/rest/index.md]: might want to make the note a aside, since its not crucial to the already somewhat complex explanation

**Comment by jpjoyal** [comment]
:+1:

File: source/rest/index.md [hunk 3]
```diff
@@ -327,6 +327,101 @@ When a webhook is created, a shared secret will be generated for it. The secret
...
+
+<api>`POST /v1/init`</api>
+
+This API is called by a mobile device or browser when the app is first loaded. It serves a number of purposes:
+
+1. Initializes a new `appUser` and `device` if they don't yet exist
+1. Update an existing app user's profile and device information
+1. Authenticate the `appUser` if `jwt` credentials are provided
+
+The API requires that a `device` be specified at the very minimum. A `userId` may also be specified to link app user accounts across devices.
```
Comment (dannytranlx) [inline source/rest/index.md]: Should we link back to http://docs.supportkit.io/#users-on-multiple-devices here?

**Comment by dannytranlx** [comment]
:maple_leaf:

File: source/rest/index.md (hunk 2 above)
Comment (alavers) [inline source/rest/index.md]: :+1:

File: source/rest/index.md (hunk 3 above)
Comment (alavers) [inline source/rest/index.md]: :+1:

**Comment by lemieux** [comment]
just do it :+1:

File: source/rest/index.md [hunk 4]
```diff
@@ -327,6 +327,105 @@ When a webhook is created, a shared secret will be generated for it. The secret
...
+| **appVersion**<br/>*optional* | A reserved string field for reporting the app version running on the device. For example: `8.0` |
+| **info**<br/>*optional* | A flat JSON structure detailing device properties. See below. |
+
+## Device Info
+
+The `device` object may also accept a flat `info` JSON object. Device information reported here will be rendered alongside app user information inside any app maker channels configured with SupportKit. The `platform` and `appVersion` fields are special in that they can be used as audience targeting criteria for whispers.
+
+| Field                     | Example                   | Relevant Platforms |
+|---------------------------|---------------------------|--------------------|
+| **platform**              | `ios`, `android`, `web`   | all                |
```
Comment (lemieux) [inline source/rest/index.md]: this.

Comment (alavers) [inline source/rest/index.md]: Woops. Good catch

**Pull request merged**